prometheus-grafana/prometheus.yml

```diff
@@ -0,0 +1,40 @@
+
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+scrape_configs:
+  - job_name: prometheus
+    scrape_interval: 10s
+    scrape_timeout: 10s
+    static_configs:
+      - targets:
+          - localhost:9090
+  - job_name: livepeer-node
+    scrape_interval: 5s
+    static_configs:
+      - targets:
+          - transcoder:7935
```

Comment:
I don't think we need to scrape the transcoder since it is not started with the `-monitor` flag and plus I don't think the transcoder logs any metrics itself.